website/blog/2023-03-02-ferretdb-building-mongodb-compatible-database.md

```diff
@@ -0,0 +1,94 @@
+---
+slug: ferretdb-building-mongodb-compatible-database
+title: "Building an Open-Source, MongoDB-Compatible Database - Our Journey to 1.0 and Beyond"
+authors: [peter]
+description: >
+    We are building FerretDB, an open source document database with MongoDB compatibility - and this is why
+image: /img/blog/ferretdb-monogdb.jpg
+tags: [document database, mongodb alternative, mongodb compatible]
+---
+
+![FerretDB - Building MongoDB compatibility](/img/blog/ferretdb-monogdb.jpg)
+
+We are building FerretDB, an open source document database with MongoDB compatibility.
+For those who are new to our story, let me explain why:
+
+<!--truncate-->
+
+[MongoDB](https://www.mongodb.com/) is a technology with excellent developer experience and very strong ecosystem.
+However, since it was abruptly [moved from a fully open source license to a proprietary one (SSPL)](https://blog.ferretdb.io/open-source-is-in-danger/), the market has been searching for an alternative.
+While there is a wide selection of open source document databases, none of them are compatible with MongoDB.
+
+This is why we founded [FerretDB](https://www.ferretdb.io/): we are on a mission to bring MongoDB database workloads back to its open source grounds.
+We are enabling [PostgreSQL](https://www.postgresql.org/) and other database backends to run MongoDB workloads, retaining the opportunities provided by the existing ecosystem around MongoDB.
+
+## So where are we now?
+
+A bit more than a year after we started working on FerretDB full time, we are a team of 9 people, working full-time and remotely from all over the world.
+We are executing on our [roadmap](https://github.com/orgs/FerretDB/projects/2), and are planning to release our 1.0 GA by the end of March.
+
+## Where are we headed?
+
+We would like to create a new standard for document databases, with [MongoDB compatibility](https://blog.ferretdb.io/mongodb-compatibility-whats-really-important/).
+FerretDB is a drop-in replacement for MongoDB, but it also aims to set a new standard which not only brings easy to use document databases back to its open source roots, but also enables different relational database engines to run document database workloads in a standardized way.
```

Comment:
```suggestion
FerretDB is a drop-in replacement for MongoDB, but it also aims to set a new standard which not only brings easy to use document databases back to its open source roots, but also enables different database engines to run document database workloads in a standardized way.
```

I would just remove the word _relational_ here, because non-relational database engines should be able to run it too.